gensim/models/poincare.py

```diff
@@ -1383,6 +1383,7 @@ def __init__(self, file_path, encoding='utf8', delimiter='\t'):
         ----------
         file_path : str
             Path to file containing a pair of nodes (a relation) per line, separated by `delimiter`.
+            To express u is v realtions, the lines should take the form `u` delimeter `v`.
```

Comment:
Typo + better formatting please. This sentence is too hard to parse.

Reply:
I believe that the line didn't have a typo. However, I am convinced it's better to use the form that you have suggested.
I have edited the docstring.
Could you please re-check it?
Thanks

Reply:
@AMR-KELEG about an example - just link to https://github.com/RaRe-Technologies/gensim/blob/master/gensim/test/test_data/poincare_hypernyms_large.tsv from docstring

Reply:
I have used the link below instead of the provided link to avoid the line length constraint.
https://github.com/RaRe-Technologies/gensim/blob/master/gensim/test/test_data/poincare_hypernyms.tsv